.github/workflows/test-integration.yml

```diff
@@ -9,64 +9,89 @@ on:
-      uses: actions/checkout@v2
-
-    - name: Start the docker images for Cypress testing
-      run: docker-compose -f docker-compose.yml -f docker-compose-ci.yml up -d
-
-    - name: Import the certificate on the Cypress docker machine
-      run: docker-compose -f docker-compose.yml -f docker-compose-ci.yml exec -T cypress sh /e2e/docker/conf/cypress_import_cert.sh
-
-    - name: Run composer
-      run: |
-        docker-compose -f docker-compose.yml -f docker-compose-ci.yml exec -T php-fpm /usr/local/bin/composer install -n --prefer-dist -o --ignore-platform-reqs
-      env:
-        SYMFONY_ENV: dev
-
-    - name: Run yarn
-      run: |
-        docker run -v ${PWD}:/var/www/html -w /var/www/html node:14 yarn install
-      env:
-        SYMFONY_ENV: dev
-
-    - name: Create front end dependencies (yarn encore production)
-      run: |
-        docker run -v ${PWD}:/var/www/html -w /var/www/html node:14 yarn run encore production
-      env:
-        SYMFONY_ENV: dev
-
-    - name: Get the database up and running
-      run: |
-        docker-compose -f docker-compose.yml -f docker-compose-ci.yml exec -T php-fpm bin/console doctrine:migrations:migrate
-      env:
-        SYMFONY_ENV: dev
-
-    - name: Remove leftover files
-      run: |
-        docker-compose -f docker-compose.yml -f docker-compose-ci.yml exec -T php-fpm /bin/rm -rf /tmp/sp-dashboard/ /tmp/sp-dashboard-sessions/ 
-
-    - name: Make the log and cache dir writable
-      run: |
-        docker-compose -f docker-compose.yml -f docker-compose-ci.yml exec -T php-fpm chmod 777 /var/www/html/var/{logs,cache}
-
-    - name: Copy parameters.yml.dist to parameters.yml
-      run: |
-        docker-compose -f docker-compose.yml -f docker-compose-ci.yml exec -T php-fpm cp /var/www/html/app/config/parameters.yml.dist /var/www/html/app/config/parameters.yml
-
-    - name: Wait until the spdashboard is accessable 
-      run: |
-        docker-compose -f docker-compose.yml -f docker-compose-ci.yml exec -T openconext sh /usr/local/sbin/spdup.sh
-    
-    - name: Disable Symfony debug mode
-      run: | 
-        docker-compose -f docker-compose.yml -f docker-compose-ci.yml exec -T php-fpm sed -i s/\'dev\',\ true/\'dev\',\ false/g /var/www/html/web/app_dev.php
-
-    - name: Run cypress tests
-      run: docker-compose -f docker-compose.yml -f docker-compose-ci.yml exec -T cypress cypress run --browser chrome
-      continue-on-error: true
-    
+      - name: Check out the repo
+        uses: actions/checkout@v2
+
+      - name: Start the docker images for Cypress testing
+        run: ${DOCKER_COMPOSE} up -d
+
+      - name: Import the certificate on the Cypress docker machine
+        run: ${DOCKER_COMPOSE_CYPRESS} sh /e2e/docker/conf/cypress_import_cert.sh
+
+      - name: Run composer
+        run: |
+          ${DOCKER_COMPOSE_PHP_FPM} /usr/local/bin/composer install -n --prefer-dist -o --ignore-platform-reqs
+        env:
+          SYMFONY_ENV: dev
```

Comment:
Can we run the application in `test` or even `prod`? That might be better than `dev`